lib/api/controllers/securityController.js

```diff
@@ -1,23 +1,48 @@
+/*
```

Comment:
It is required by the Apache 2.0 license. And a PR is already underway, I'm updating files in this PR to ease up future merges.